index.ts

```diff
@@ -117,11 +117,16 @@ async function showInfo(showOnDevice?: boolean) {
   const accountIdentifier = AccountIdentifier.fromPrincipal({
     principal: identity.getPrincipal(),
   });
+  const bufferKey = identity.getPublicKey() as Secp256k1PublicKey;
+  const hexPubKey = buf2hex(bufferKey.toRaw());
```

Comment:
Would it make sense to add a `Secp256k1PublicKey.toHex()` method that encapsulates this?

Then here the code changes would be:

```
const publicKey = identity.getPublicKey() as Secp256k1PublicKey;

...

log(
    chalk.bold('Public key: ') + publicKey.toHex()
)
```

Reply:
yes, good point.

Reply:
done. I also added the checks there and used it in the claimNeurons function.